nixos/modules/services/misc/radarr.nix

```diff
@@ -11,6 +11,15 @@ in
     services.radarr = {
       enable = mkEnableOption "Radarr";
 
+      package = mkOption {
+        description = "Radarr package to use";
+        default = pkgs.radarr;
+        defaultText = literalExpression ''
+          pkgs.radarr
+        '';
```

Comment:
```suggestion
        example = literalExpression "pkgs.radarr";
```

Reply:
applied in 42adaed31776b6fbddd2a162689e455d09a455a6